### DISCUSSION TITLE
[Suggestion] Ability to share files using app-based hotspot

### DISCUSSION THREAD

**Discussion by user1823** [localsend/localsend#1147] [Discussion (English, International)]
Many times, I have to share files with others when we are not connected to WiFi. I can use LocalSend to share the files by first creating a hotspot and then asking the other person to join it.

However, sharing files would be much more convenient if the app was able to create its own hotspot just like Nearby Share (Android) or AirDrop does it.

**Comment by SweetnsourOfficial** [comment]
Wifi direct right?

**Comment by SweetnsourOfficial** [comment]
I had the same suggestion 😊
